src/main/java/uk/ac/bath/se/DungeonView.java

```diff
@@ -14,7 +14,7 @@
     private boolean collided = false;
     Dungeon dungeon;
 
-    public DungeonView() {
+    public DungeonView(String string) {
```

Comment:
I'm not sure about passing it in this way but for now could you rename it playerName or something more descriptive please

Reply:
Yeah. I can do it.